core/tests/protractor/subscriptions.js

```diff
@@ -72,35 +71,25 @@ describe('Subscriptions functionality', function() {
     element(by.css('.protractor-test-subscription-button')).click();
     browser.waitForAngular();
     element(by.css('.protractor-test-subscription-button')).click();
-    browser.get(general.USER_PREFERENCES_URL);
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).count()).toEqual(1);
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).first().getText()).toMatch(
-      'creator...');
+    preferencesPage.get();
+    preferencesPage.expectSubscriberCountToEqual(1);
+    preferencesPage.expectFirstSubscriberToBe('creator...');
     users.logout();
 
     users.login('creator1@subscriptions.com');
     creatorDashboardPage.get();
     browser.waitForAngular();
     creatorDashboardPage.navigateToSubscriptionDashboard();
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).first().getText()).toMatch(
-      'learner...');
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).last().getText()).toMatch(
-      'learner...');
+    preferencesPage.expectFirstSubscriberToBe('creator...');
+    preferencesPage.expectLasttSubscriberToBe('creator...');
     users.logout();
 
     users.login('creator2@subscriptions.com');
     creatorDashboardPage.get();
     browser.waitForAngular();
     creatorDashboardPage.navigateToSubscriptionDashboard();
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).count()).toEqual(1);
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).last().getText()).toMatch(
-      'learner...');
+    preferencesPage.expectSubscriberCountToEqual(1);
+    preferencesPage.expectLasttSubscriberToBe('creator...');
```

Comment:
Lastt is misspelled. This should have failed locally. Did you run the tests locally before submitting? If not, please do so in the future.

Reply:
okay